src/app/posthog-provider.jsx

```diff
@@ -0,0 +1,31 @@
+'use client';
+
+import posthog from 'posthog-js';
+import { PostHogProvider as PostHogProviderBase } from 'posthog-js/react';
+import PropTypes from 'prop-types';
+import { useEffect } from 'react';
+
+// Provider configured based by the official Posthog documentation:
+// https://posthog.com/docs/libraries/next-js
+
+const PostHogProvider = ({ children }) => {
+  useEffect(() => {
+    const key = process.env.NEXT_PUBLIC_POSTHOG_KEY;
+
+    if (key) {
+      posthog.init(key, {
```

Comment:
let's use the same proxy hosts we use in console? 
https://github.com/neondatabase/cloud/blob/main/web/analytics/analytics/provider.tsx#L35

Reply:
I even don't access to our segment account anymore.

Reply:
oh, stupid me, sorry)